ActivityMerger/templates/download.html

```diff
@@ -0,0 +1,63 @@
+{% extends "base.html" %}
+
+{% block body %}
+<div class="row">
+
+    <div class="jumbotron">
+        <h2 class="sub-header">IATI Activity Merge Tool</h2>
+        <p class="lead">Merge multiple IATI activity files</p>
+    </div>
+
+    <div class="panel-group" id="accordionOutput" role="tablist" aria-multiselectable="true">
+        <div class="panel panel-default">
+            <div class="panel-heading" role="tab" id="headingOne" data-toggle="collapse" data-parent="#accordionOutput"
+                data-target="#collapseOutput" aria-expanded="true" aria-controls="collapseOutput">
+                <h4 class="panel-title">
+                    <a class="accordion-toggle">
+                        <span class="glyphicon glyphicon-download" aria-hidden="true"></span>Download merged activity file
+                    </a>
+                </h4>
+            </div>
+            <div id="collapseOutput" class="panel-collapse collapse in" role="tabpanel" aria-labelledby="headingOne">
+                <div class="panel-body">
+                    <div><label>The merged file was generated at: {{ timestamp }}</label></div>
+                    <div><label>This is composed of {{ session['files']|length }} activity files.</label></div>
```

Comment:
I suggest that we re-write this along the lines of 

"This is composed of {{ session['files']|length }} activity files and it contains {{ session['totalActivities'] }} activities (XML elements)."